mesa/mesaPROTON_OC.py

```diff
@@ -914,7 +920,7 @@ def init_professional_links(self):
                 conn_pool = list(self.rng.choice(list(total_pool), conn, replace=False))
                 employee.makeProfessionalLinks(conn_pool)
 
-    def df_to_lists(self,df):
+    def df_to_lists(self,df, split_row=True):
         """
         This function transforms a pandas DataFrame into nested lists as follows:
```

Comment:
this should be updated to reflect split_row; also mention that the column names are examples?